api/v1alpha1/common.go

```diff
@@ -51,7 +51,7 @@ type Level struct {
 	// Multiple paths should be separated with comma. For example: "/mnt/cache1,/mnt/cache2".
 	// +kubebuilder:validation:MinLength=1
 	// +required
-	Path string `json:"path,omitempty"`
+	Path string `json:"path"`
```

Comment:
I think it's optional for some runtime, for example vineyard.

Reply:
Fixed.